### DISCUSSION TITLE
"devui-theme"没有导出的成员“infinityTheme“

### DISCUSSION THREAD

**Discussion by rd-nolan** [DevCloudFE/vue-devui#1528] [General]
<img width="612" alt="image" src="https://user-images.githubusercontent.com/35715627/211445492-e3a143f7-ea02-4879-ab44-5e79f67a8f0a.png">

**Comment by kagol** [comment]
@rd-carter 你是在什么场景下使用的呢？可否详细描述下？

通过以下方式是可以导入`infinityTheme`主题的：
```
import { infinityTheme } from 'devui-theme'
```

**Reply by rd-nolan** [reply]
模块“"devui-theme"”没有导出的成员“infinityTheme”

**Reply by rd-nolan** [reply]
@kagol "devui-theme": "^0.0.4",有没有可能是版本问题

**Comment by taunt123** [comment]
![21faeae7f12f8e3aba8df64591016e3](https://user-images.githubusercontent.com/80139344/230314784-2dab20cf-0a2a-44e2-a8a0-2c8fbf65a3e5.png)
